docs/sources/get-started/ride-share-tutorial.md

```diff
@@ -0,0 +1,297 @@
+
+In this scenario a load generator will send mock-load to the 3 servers as well as their respective endpoints. This will allow us to see how the application is performing per region and per vehicle type.
+
+{{<docs/ignore>}}
+{{< admonition type="tip" >}}
+A setup script is running in the background to install the necessary dependencies. This should take no longer than 30 seconds. Your instance will be ready to use once you `Setup complete. You may now begin the tutorial`.
+{{< /admonition >}}
+{{</docs/ignore>}}
+
+<!-- INTERACTIVE page intro.md END -->
+
+<!-- INTERACTIVE page step1.md START -->
+
+## Clone the repository
+Clone the repository to your local machine:
+
+```bash
+git clone https://github.com/grafana/pyroscope.git && cd pyroscope
+```
+
+Navigate to the tutorial directory:
+
+```bash
+cd examples/language-sdk-instrumentation/python/rideshare/flask
+```
+
+## Start the application
+
+Start the application using Docker Compose:
+
+```bash
+docker compose up -d
+```
+
+This may take a few minutes to download the required images and build the demo application. Once ready, you will see the following output:
+
+```console
+ ✔ Network flask_default  Created
+ ✔ Container flask-ap-south-1  Started
+ ✔ Container flask-grafana-1  Started
+ ✔ Container flask-pyroscope-1  Started     
+ ✔ Container flask-load-generator-1 Started 
+ ✔ Container flask-eu-north-1 Started       
+ ✔ Container flask-us-east-1 Started  
+```
+
+(Optional) To verify the containers are running run:
+
+```bash
+docker ps -a
+```
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Accessing the Pyroscope UI
+
+Pyroscope includes a web-based UI that you can use to view the profile data. To access the Pyroscope UI, open a browser and navigate to [http://localhost:4040](http://localhost:4040).
+
+### How tagging works
+
+In this example, the application is instrumented with Pyroscope using the Python SDK. The SDK allows you to tag functions with metadata that can be used to filter and group the profile data in the Pyroscope UI. In this example we have used two forms of tagging; static and dynamic.
+
+To start lets take a look at a static tag use case. Within the `server.py` file we can find the Pyroscope configuration:
+
+```python
+pyroscope.configure(
+	application_name = app_name,
+	server_address   = server_addr,
+    basic_auth_username = basic_auth_username, # for grafana cloud
+    basic_auth_password = basic_auth_password, # for grafana cloud
+	tags             = {
+        "region":   f'{os.getenv("REGION")}',
+	}
+)
+```
+The reason this tag is considered static is due to the fact that the tag is set at the start of the application and doesn't change. In our case this is useful for grouping profiles on a per region basis. Allowing us to see the performance of the application per region.
```

Comment:
```suggestion
This tag is considered static is because the tag is set at the start of the application and doesn't change.
In this case, it's useful for grouping profiles on a per region basis, which lets yousee the performance of the application per region.
```